inference-engine/thirdparty/clDNN/api/loop.hpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+#include <vector>
+#include <functional>
+#include "primitive.hpp"
+#include "topology.hpp"
+
+#define DEFAULT_MAX_NUM_ITERATION 128
+namespace cldnn {
+/// @addtogroup cpp_api C++ API
+/// @{
+/// @addtogroup cpp_topology Network Topology
+/// @{
+/// @addtogroup cpp_primitives Primitives
+/// @{
+///
+/// @brief Adds primitive which performs recurrent execution of the topology.
+///
+/// @details
+/// @n   The body topology for recurrent execution is described in the body
+/// @n   The execution of the body topology iterates through the data in the given axis.
```

Comment:
Please add this comment:
"Note that only loops with fixed iteration count is being validated and supported currently."

Reply:
fixed